src/Core/Houston.vala

```diff
@@ -17,85 +17,154 @@
 */
 
 public class AppCenterCore.Houston : Object {
+    public signal void refreshed ();
+
     private const string HOUSTON_API_URL = "https://developer.elementary.io/api";
+    private const string NEWEST_CATEGORY = "Newest";
+    private const string UPDATED_CATEGORY = "Updated";
+    private const string APPS_KEY = "apps";
 
     private Soup.Session session;
+    private GLib.KeyFile cache_file = null;
```

Comment:
If the `cache_file` is nullable, it should contain `?` at the end of the type.
I guess the `KeyFile` could also be static here?